### DISCUSSION TITLE
SONOFF SNZB-02 - no data shown. Please advise!

### DISCUSSION THREAD

**Discussion by pauliukiukas** [Koenkk/zigbee2mqtt#15561] [Devices]
Dear Experts! 

Failing finding the reason why the temperature/humidity sensor from SONOFF is recognized, but no metrics (temperature, humidity etc.) visible in Z2M (nor in HomeAssistant). 
I have tried with 5-6 different identical sensors- the same. Removing and adding again - no change. 
I am pretty sure that I miss something in configuring, so any advice is very welcome. 

Coordinator - SONOFF 3.0 Dongle Plus (that was used with ZHA and all metrics were shown).

![Capture3](https://user-images.githubusercontent.com/98475534/207817645-5d0147a9-d5fa-4d26-9195-65e7c0a3c436.PNG)
![Capture2](https://user-images.githubusercontent.com/98475534/207817650-988cd9b2-1b9a-4f16-823e-43fc6ad9745f.PNG)
![Capture1](https://user-images.githubusercontent.com/98475534/207817654-d0ddc679-b658-431d-80c7-80f41ce65993.PNG)

**Comment by aknifeforfruit** [comment]
这是来自QQ邮箱的自动回复邮件。您好，您的邮件已收到，谢谢！

**Comment by undefinedv** [comment]
got it . I'll get back to you later.
